build/dist

```diff
@@ -144,14 +152,17 @@ HIVE_VERSION=$("$MVN" help:evaluate -Dexpression=hive.version $@ 2>/dev/null\
     | grep -v "WARNING"\
     | tail -n 1)
 
-echo "Building Kyuubi package of version $VERSION against Spark version - $SPARK_VERSION"
+echo "Building Kyuubi package of version $VERSION against Flink $FLINK_VERSION, Spark $SPARK_VERSION"
 
-SUFFIX="-$NAME"
-if [[ "$NAME" == "none" ]]; then
-  if [[ "$SPARK_PROVIDED" == "true" ]]; then
-    SUFFIX=""
-  else
-    SUFFIX="-spark-${SPARK_VERSION:0:3}-hadoop${SPARK_HADOOP_VERSION}"
+if [[ "$NAME" != "none" ]]; then
```

Comment:
`build/dist --tgz --flink-provided --spark-provided` => `apache-kyuubi-1.5.0-SNAPSHOT-bin.tgz`
`build/dist --tgz --name custom` => `apache-kyuubi-1.5.0-SNAPSHOT-bin-custom.tgz`
`build/dist --tgz --spark-provided` => `apache-kyuubi-1.5.0-SNAPSHOT-bin-flink-1.14.tgz`
`build/dist --tgz --flink-provided` => `apache-kyuubi-1.5.0-SNAPSHOT-spark-3.1-hadoop3.2.tgz`
`build/dist --tgz` => `apache-kyuubi-1.5.0-SNAPSHOT-bin-flink-1.14-spark-3.1-hadoop3.2.tgz`

Reply:
Does this name pattern looks good?

Reply:
currently, I am fine with the naming. But, if we introduce trino/presto, the name would be very long.

Reply:
not good, we have a release file already

Reply:
any suggestions?

Reply:
> not good, we have a release file already

We only have one apache official release binary tarball `apache-kyuubi-{VERSION}-bin.tgz`

Reply:
let's leave it as-is, we don't change the naming policy with a new engine added

Reply:
reverted